x/uibc/quota/keeper/msg_server.go

```diff
@@ -13,22 +13,21 @@ type msgServer struct {
 	kb Builder
 }
 
-// NewMsgServerImpl returns an implementation of MsgServer for the x/uibc
-// module.
+// NewMsgServerImpl returns an implementation of uibc.MsgServer
 func NewMsgServerImpl(kb Builder) uibc.MsgServer {
 	return &msgServer{kb: kb}
 }
 
 // GovUpdateQuota implements types.MsgServer
-func (m msgServer) GovUpdateQuota(goCtx context.Context, msg *uibc.MsgGovUpdateQuota) (
+func (m msgServer) GovUpdateQuota(ctx context.Context, msg *uibc.MsgGovUpdateQuota) (
```

Comment:
This name change might. be better left reverted, since `ctx` everywhere else in our code refers to an sdk context